components/data/datatable/mixins/custom-template.js

```diff
@@ -1,7 +1,7 @@
 export default {
   methods: {
     hasCustomTemplate: function (col) {
-      return Boolean(this.$scopedSlots[col.prop])
+      return Boolean(this.$scopedSlots[col.prop] || this.$scopedSlots[col.slot])
```

Comment:
Actually you don't need to keep `col.prop`, you can change that to `col.slot` as you can see [here](https://github.com/oswaldofreitas/vulma/blob/master/components/data/datatable/mixins/columns.js#L20) `prop` is used if `slot` wasn't specified. `cols` is used [here](https://github.com/oswaldofreitas/vulma/blob/master/components/data/datatable/mixins/fixed-columns.js#L11) by `scrollableColumns` for example and which in turn is used [here](https://github.com/oswaldofreitas/vulma/blob/master/components/data/datatable/datatable.vue#L75)

Reply:
so, by the end you'll have:
`return Boolean(this.$scopedSlots[col.slot])`